projects/WTI-API/src/main/controllers/ContestController.java

```diff
@@ -547,9 +561,11 @@ public Response getStandings(
 			synchronized (updateStandingsMutex) {
 				
 				// check if some event has occurred which could have changed the standings
+				wtiServerStandingsAreCurrent = false; // NEW - forcing to false so will recalculate for each user
```

Comment:
This is incredibly inefficient -- to the point where in some contests (like in ACPC) where there are thousands of users (yet no Divisions) this might completely overload the system.  I think forcing this variable false should at least be conditional on whether or not the .ini file contains `server.wtiBoardUseDivision`.

Reply:
On further review of code below, I see there IS a test for whether server.wtiBoardUseDivision is present in the .ini file.  I think that check should be done here as well...

Reply:
I was concerned with that force execution also,  I agree it should be based on server.wtiBoardUseDivision.

Reply:
Adding conditional around wtiServerStandingsAreCurrent = false;

Reply:
> This is incredibly inefficient --